src/CampaignsAdmin/NewCampaignForm.tsx

```diff
@@ -68,12 +68,12 @@ export const NewCampaignForm = () => {
             }}
           ></Input>
           <Button
-            type="primary"
```

Comment:
String literals as HTML props should be in double quotes. Everything else in singles :P